src/file-uploader/file-uploader.js

```diff
@@ -167,33 +169,25 @@ function FileUploader(props: PropsT) {
                         </ContentMessage>
                       )}
                       {props.errorMessage ? (
-                        <Button
+                        <ButtonComponent
                           kind={KIND.minimal}
                           onClick={() => {
                             props.onRetry && props.onRetry();
                           }}
-                          overrides={{
```

Comment:
can you maintain this prop? same thing on L189. Other than this LGTM. thanks!

Reply:
Sorry, I missed these two, it should be fine now. Thanks!